packages/airnode-node/src/providers/state.ts

```diff
@@ -33,6 +33,11 @@ export function buildEVMState(
     logFormat: config.nodeSettings.logFormat,
     logLevel: config.nodeSettings.logLevel,
     minConfirmations: chain.minConfirmations || BLOCK_MIN_CONFIRMATIONS,
+    // If the _minConfirmations reserved parameter is set for one or more endpoints,
+    // a request may override minConfirmations, but we don't know if it will or the value
+    // until we actually fetch the requests, perform API calls, and extract reserved
+    // parameter overrides in processSuccessfulApiCall
+    mayOverrideMinConfirmations: checkForMinConfirmationsReservedParam(config),
```

Comment:
@amarthadan and @Siegrift - this comment and new boolean parameter summarize the challenge and my approach. If `mayOverrideMinConfirmations` is `true`, we have to assume `minConfirmations` is set to zero otherwise we wouldn't fetch all of the requests which might specify a value for `_minConfirmations` (see my update to the `fetch` function within `packages/airnode-node/src/evm/requests/event-logs.ts`). Would love your feedback 🙏

Reply:
Hmm, I was thinking whether it even makes sense to distinguish these two situations and whether it wouldn't be easier to fetch them always all. But looking at [this table](https://docs.api3.org/qrng/reference/chains.html) it looks like it may be worth it. So, in that case, this approach makes sense to me :+1: